quartz/plugins/emitters/componentResources.ts

```diff
@@ -149,9 +150,10 @@ function addGlobalPageResources(
       loadTime: "afterDOMReady",
       contentType: "inline",
       script: `
-        const socket = new WebSocket('${wsUrl}')
-        socket.addEventListener('message', () => document.location.reload())
-      `,
+          const socket = new WebSocket('${wsUrl}')
+          // reload(true) ensures resources like images and scripts are fetched again
```

Comment:
TIL! seems to only be supported for firefox though

Reply:
Updated comment to say only for firefox :)